### PR TITLE
Created url shortening application

### PR DESCRIPTION
**Pull request by tuba-mouqeem**
Please review and suggest, how can I make it better?

### CODE REVIEW

**Comment by darknao** [comment]
I'm not sure how to review this...
Can you explain what you have in mind? or provide a few examples on how to use this function?
I see a sqlite database, but you never use it to store anything. Did I miss something? Is this code working?

File: src/mote/url_shortner.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        else:
+            self.Storing_UrlId[longUrl] = self.id
+            shorten_url = self.convereter_func(self.id)
+            # Counter increment
+            self.id = self.id + 1
+
+        return "meetbot.fedoraproject.org/" + shorten_url
+
+    # Converting b10 to b62(Alphanumeric)
+    def convereter_func(self, id):
```
Comment (Zlopez) [inline src/mote/url_shortner.py:43]:
```suggestion
    def converter_func(self, id):
```

**Review by darknao — CHANGES_REQUESTED** [review]
There are some good ideas here, but the implementation is far from complete.
Possible improvements are:
- Create 2 functions, one to convert a long url to a short one, and another to do the reverse (short to long url)
- The first function should check if the url already exist in db, and if not, create a new ShortUrl instance, generate the short url, save it into the db, then return the short url.
- The second one should query the db for the desired short url, and return the corresponding long one.
- Combine the 2 classes into one: move `converter_func` to the ShortUrl class, and rename it to something more meaningful like `get_short_url`. This function should use the object `id` (available as `self.id`)
- Create a route endpoint that take a short url in parameter, and redirect to the matching long url (something like `/<short_url>` or `/s/<short_url>`)

**Comment by t0xic0der** [comment]
Closing this as stale

**Pull request closed without merging**